ag/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# Maintainer: Mihail Konev <k.mvc@ya.ru>
+
+pkgname=ag
+srcname=the_silver_searcher
+pkgver=git
```

Comment:
A fixed `pkgver` disables the ability to update this package (via pacman) in the future.
See https://aur.archlinux.org/packages/?O=0&K=-git for a better `pkgver` scheme.

Also if you have time: 
https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/the_silver_searcher
https://wiki.archlinux.org/index.php/Arch_packaging_standards
https://wiki.archlinux.org/index.php/PKGBUILD